doc/source/tutorials/go_models.rst

```diff
@@ -0,0 +1,133 @@
+=========
+Go models
+=========
+
+GoMartini is a popular method for retaining the secondary and tertiary structures of proteins originating from the lab
+of `Adolfo Poma <https://pubs.acs.org/doi/full/10.1021/acs.jctc.6b00986>`_. In contrast to an elastic network, the Go
```

Comment:
This might be misunderstood. Go models have been around for a long time, but I think it is reasonable to say Adolfo was the first to apply to M3. However, might be worth checking this fact with Siewert.